changes/pr4059.yaml

```diff
@@ -0,0 +1,2 @@
+enhancement:
+  - "Add `access_token_secret` to `GitHub`, `GitLab`, and `Bitbucket` storage, making the Prefect secret containing an access token to these services configurable - [#4059](https://github.com/PrefectHQ/prefect/pull/4059)"
```

Comment:
Even though they're mainly private methods and probably weren't accessed I think it's worth calling out the removal of the get_env_runner functions and that the git utilities was absorbed by the respective storage classes.